deploy/kubernetes/base/controller-deployment.yaml

```diff
@@ -30,25 +30,14 @@ spec:
           tolerationSeconds: 300
       containers:
         - name: file-cache-plugin
-          image: 648284600874.dkr.ecr.us-east-1.amazonaws.com/aws-file-cache-csi-driver:v0.0.1
-          args :
+          image: 651262257939.dkr.ecr.us-east-1.amazonaws.com/file-cache-dynamic-provisioning:v0.0.20
+          imagePullPolicy: IfNotPresent
+          args:
             - --endpoint=$(CSI_ENDPOINT)
             - --v=5
           env:
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
-            - name: AWS_ACCESS_KEY_ID
-              valueFrom:
-                secretKeyRef:
-                  name: aws-secret
-                  key: key_id
-                  optional: true
-            - name: AWS_SECRET_ACCESS_KEY
-              valueFrom:
-                secretKeyRef:
-                  name: aws-secret
-                  key: access_key
-                  optional: true
```

Comment:
I think we took this out as part of debugging but let's add them back for when we go back to testing the sts issue